src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1127,6 +1128,15 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
                     _terminal->SetSelectionEnd(terminalPosition, mode);
                     _selectionNeedsToBeCopied = true;
                 }
+                if (ctrlEnabled && multiClickMapper == 1)

```

Comment:
we may need to add some mutual exclusion cases here -- right now "ctrl+shift+click" will end up in the selection update case above

Reply:
Plus, what happens when a user is in VT mouse mode, and wants to click a link? They need shift to ignore VTMM, but then I'd think the ctrl+click for the hyperlink should have higher precedence (if there's a link under the cursor)